pym/bob/cmds/misc.py

```diff
@@ -77,21 +78,14 @@ def showPrefixed(packages, recurse, showAll, stack, level=0):
     recipes.parse()
 
     showAll = args.all
-    roots = sorted(recipes.generatePackages(lambda s,m: "unused", sandboxEnabled=args.sandbox).values(), key=lambda a:a.getName())
+    roots = collections.OrderedDict(sorted(recipes.generatePackages(lambda s,m: "unused", sandboxEnabled=args.sandbox).items()))
```

Comment:
Due to the `sorted()` above this is unneeded. Just `recipes.generatePackages(...`...